### ISSUE TITLE
cannot filter on boolean type

### ISSUE THREAD

**Issue by wpcfan** [typicode/json-server#412]
database looks like below, and http://localhost:3000/todos/?completed=true is working, but http://localhost:3000/todos/?completed=false gets an empty array. I am using 0.9.0

```
{
  "todos": [
    {
      "id": "bf75769b-4810-64e9-d154-418ff2dbf55e",
      "desc": "getting up",
      "completed": false,
      "userId": "1"
    },
    {
      "id": "5894a12f-dae1-5ab0-5761-1371ba4f703e",
      "desc": "have breakfast",
      "completed": true,
      "userId": "2"
    },
    {
      "id": "0d2596c4-216b-df3d-1608-633899c5a549",
      "desc": "go to school",
      "completed": true,
      "userId": "1"
    },
    {
      "id": "0b1f6614-1def-3346-f070-d6d39c02d6b7",
      "desc": "test",
      "completed": false,
      "userId": "2"
    },
    {
      "id": "c1e02a43-6364-5515-1652-a772f0fab7b3",
      "desc": "This is a te",
      "completed": false,
      "userId": "1"
    }
  ],
  "users": [
    {
      "id": "1",
      "username": "wang",
      "password": "1234"
    },
    {
      "id": "2",
      "username": "peng",
      "password": "5678"
    }
  ]
}
```

**Comment by jantrienes** [comment]
I were able to reproduce this behaviour. It seems to be an issue with the filtering. 

There is a conditional check within `/server/router/plural.js` that matches for `false` values and thus excludes the instance from the result list. 

**See:**
https://github.com/typicode/json-server/blob/b5eec153f7ca9551338e3b884c0d140f93cde42e/src/server/router/plural.js#L123

I assume that this check should catch `undefined` values and exclude them from the filtering. Therefore, replacing it with the following should do the trick. 

```js
if (elementValue == null) {
  return
}
```

**Comment by typicode** [comment]
Should be good in `v0.9.1` thanks to @jantrienes PR

**Comment by Jaoow** [comment]
Although the issue is closed, I fell into the same problem. To make "true" or "false" work, replace it with "1" (for true) and "0" (for false).

As in the example: 

> `http://localhost:3000/todos/?completed=true`

should be:

> `http://localhost:3000/todos/?completed=1`

**Comment by typicode** [comment]
Fixed in v1.0.0-beta.0 ;)
